cluster/vagrant/setup_kubernetes_master.sh

```diff
@@ -50,6 +50,29 @@ else
   kubectl create -f kube-$NETWORK_PROVIDER.yaml
 fi
 
+# Create an ingress-nginx setup
+# See: https://github.com/kubernetes/ingress-nginx/tree/master/deploy
+{
+  curl_yaml_combine() { for URL in $@; do curl -L $URL ; echo --- ; done }
+
+  # Do the basic deployment
+  curl_yaml_combine \
```

Comment:
weave's manifest is versioned. I'm trying to avoid a situation where our tests all start breaking because these ingress manifests change. When that happens we waste a day trying to unblock everyone rather than updating on our own terms.

If there's no version number in there, we really don't have any guarantees about stability.

With weave we're at least using something that resembles a release. 
```
kubectl apply -f https://github.com/weaveworks/weave/releases/download/v1.9.4/weave-daemonset-k8s-1.6.yaml
```

Reply:
Ah good point. ket me try something

Reply:
I can't use a previous tag, as there were significant changes. But there will be a release before the end fo week (let's see in what TZ), then I will use versioned files.